app/src/Liquidator.ts

```diff
@@ -0,0 +1,233 @@
+import Web3 from "web3";
+import { Log } from "web3-core";
+import { Contract } from "web3-eth-contract";
+import { AbiItem } from "web3-utils";
+import LiquidatorABIJson from "./abis/Liquidator.json";
+import TxManager from "./TxManager";
+import winston from "winston";
+import Bottleneck from "bottleneck";
+
+const FACTORY_ADDRESS: string = process.env.FACTORY_ADDRESS!;
+const CREATE_ACCOUNT_TOPIC_ID: string = process.env.CREATE_ACCOUNT_TOPIC_ID!;
+const WALLET_ADDRESS = process.env.WALLET_ADDRESS!;
+const ALOE_INITIAL_DEPLOY = 0;
+const POLLING_INTERVAL = 60_000;
+const STATUS_HEALTHY = 200;
+const STATUS_NOT_HEALTHY = 503;
+const REQUEST_RATE_LIMIT = 250;
```

Comment:
Units? Based on the [docs](https://www.npmjs.com/package/bottleneck#schedule), perhaps `REQUEST_RATE_LIMIT_IN_MS`?